src/Microsoft.ML.StaticPipe/LpNormalizerStaticExtensions.cs

```diff
@@ -16,7 +16,7 @@ private sealed class OutPipelineColumn : Vector<float>
         {
             public readonly Vector<float> Input;
 
-            public OutPipelineColumn(Vector<float> input, LpNormalizingEstimatorBase.NormalizerKind normKind, bool subMean)
+            public OutPipelineColumn(Vector<float> input, LpNormalizingEstimatorBase.NormKind normKind, bool subMean)
```

Comment:
>NormKind [](start = 85, length = 8)

this feels backwards, from longer, more descriptive name to abbreviation. 

NormalizationKind ?